--- E:/code/cpeject/test_mooc-clin/DirModeEx/tests/demo_proj/src/demo.c
+++ E:/code/cpeject/test_mooc-clin/DirModeEx/tests/demo_proj/src/demo.c
@@ -117,9 +117,8 @@
-    { "var_maxtx1[]", "Exam4", "Exam4", "Exam4", "Exam、、nple8", NULL },
+    { "Exam4", "Exam4", "Exam4", "Exam、、nple8", "Exam4", NULL }
-    { "var_maxtx1[]", "Exam4", "Exam4", "Exam4", "Exam、、nple8", NULL }
+};
-};
+
-
+static const _Tr_TEXT var_maxtx2[22322] = {
-static const _Tr_TEXT var_maxtx2[22322] = {
+{ "中文1", "Exam4", "Exam4", "Exam4", "Exam、、nple8", NULL },
-{ "中文1", "Exam4", "Exam4", "Exam4", "Exam、、nple8", NULL },
+{ "中文2", "Exam4", "Exam4", "Exam4", "Exam、、nple8", NULL },
-{ "中文2", "Exam4", "Exam4", "Exam4", "Exam、、nple8", NULL },
+{ "中文3", "Exam4", "Exam4", "Exam4", "Exam、、nple8", NULL },
-{ "中文3", "Exam4", "Exam4", "Exam4", "Exam、、nple8", NULL },
+};
-};
--- E:/code/cpeject/test_mooc-clin/DirModeEx/tests/demo_proj/src/demo.c
+++ E:/code/cpeject/test_mooc-clin/DirModeEx/tests/demo_proj/src/demo.c
@@ -121,4 +121,3 @@
-{ "中文1", "Exam4", "Exam4", "Exam4", "Exam、、nple8", NULL },
+{ "中文2", "Exam4", "Exam4", "Exam4", "Exam、、nple8", "Exam4", NULL },
-{ "中文2", "Exam4", "Exam4", "Exam4", "Exam、、nple8", NULL },
+{ "中文3", "Exam4", "Exam4", "Exam4", "Exam、、nple8", "Exam4", NULL }
-{ "中文3", "Exam4", "Exam4", "Exam4", "Exam、、nple8", NULL },
+};
-};
